tools/install/Makefile.skel

```diff
@@ -76,29 +76,33 @@ FSHAREDFLAGS =
 LIBDIR       =
 endif
 
+# set rpath flag
 ifeq ($(OS),Linux)
 RPATH_OPTION = -R
 else
 RPATH_OPTION = -rpath
 endif
 
+# other compiler flags
+LDFLAGS = -L$(CVODELIB) -L$(OPENLIBMDIR) -Wl,$(RPATH_OPTION),$(LIBDIR):$(OPENLIBMDIR) -lopenlibm -lsundials_fcvode -lsundials_cvode -lsundials_fnvecserial -lsundials_nvecserial -ldl
+
+# directories
 OBJ = obj
 SRC = src
 UNITTESTDIR = travis/unit_tests
 
+# executable
 AOUT = atchem2
 
-all: $(AOUT)
-
+# 
```

Comment:
UNITTEST_SRCS contains all source files except atchem.f90, as atchem.f90 is not unit-tested.